### DISCUSSION TITLE
MiEX Importer plugin for Blender.

### DISCUSSION THREAD

**Discussion by xboxer214** [BramStoutProductions/MiEx#19] [Ideas]
I think it would be nice to have an addon/plugin for Blender that let's us directly import the MiEx file types, so we don't have to deal with Blenders lack of full USD support.

**Comment by zNightlord** [comment]
As someone helping with MCprep addon developing things, I highly against with this idea.

- You introduce a complex file format structure while something already better like USD exists
- Do you know how Blender with Python I/O works? 
*For this see the legacy OBJ importer made with Python and the new C++ OBJ importer contributed by the legendary Aras, for Minecraft example see Igrium's exporter*
- And finally I don't want to see someone no purpose suffers in making a custom importer, well it is sucks ngl.

**Comment by zNightlord** [comment]
USD in Blender right now is fine in a basic way, import export, thank to Nvidia. MaterialX is being work on with the help of AMD devs so.

There is no head developer in charge of directing USD related works, they have to move back and forward to what they need to prioritize and USD to review contributed code by them.
